src/Driver/Driver.jl

```diff
@@ -89,64 +43,76 @@ end
     end
 end
 
+_init_array(::Type{Array}) = nothing
+
+const cuarray_pkgid =
+    Base.PkgId(Base.UUID("3a865a2d-5b23-5a0f-bc46-62713ec82fae"), "CuArrays")
 function gpu_allowscalar(val)
     if haskey(Base.loaded_modules, CLIMA.cuarray_pkgid)
         Base.loaded_modules[CLIMA.cuarray_pkgid].allowscalar(val)
     end
     return nothing
 end
 
+# Note that the initial values specified here are overwritten by the
+# command line argument defaults in `parse_commandline()`.
+Base.@kwdef mutable struct CLIMA_Settings
+    disable_gpu::Bool = false
+    show_updates::String = "60secs"
```

Comment:
I'm not really a fan of having the number and units combined into one string (now we have to parse this and remember the format).

Reply:
I see your point. But command line arguments are tricky. `ArgParse.jl` cannot, I think, handle multiple arguments (e.g. `--checkpoint 60 secs`) so you would have to pass `--checkpoint "60 secs"` which is a bit ugly to me.

Reply:
What about
`show_update_frequency::Int`
`show_update_units::String`
?

Reply:
So, avoiding the need to have two or more command line arguments for each option is one of the reasons for this PR -- it is already pretty busy. I think if we have a consistent definition for these intervals it should be fine. Let's discuss further at the next meeting.

Reply:
@simonbyrne: any thoughts on this?

Reply:
I agree with @kpamnany that it would be good to minimise the number of arguments.